src/extension.js

```diff
@@ -91,6 +91,7 @@ export default class SensorsExtension
         this._iconTemp = Gio.icon_new_for_string(this.path + '/icons/hicolor/scalable/status/sensors-temperature-symbolic.svg');
         this._iconFan = Gio.icon_new_for_string(this.path + '/icons/hicolor/scalable/status/sensors-fan-symbolic.svg');
         this._iconVolt = Gio.icon_new_for_string(this.path + '/icons/hicolor/scalable/status/sensors-voltage-symbolic.svg');
+        this._iconWatt = Gio.icon_new_for_string(this.path + '/icons/hicolor/scalable/status/sensors-voltage-symbolic.svg');
```

Comment:
Er, better do an icon for watts. Maybe just the voltage one with a "W" (the common symbol for electrical power in Watts) over it.

Reply:
Re-wrote the old voltage icon using a simple poly line, which renders cleaner. Used this new version as a basis for the wattage icon.